python/pyspark/ml/util.py

```diff
@@ -78,7 +78,14 @@ def overwrite(self):
         raise NotImplementedError("MLWriter is not yet implemented for type: %s" % type(self))
 
     def context(self, sqlContext):
-        """Sets the SQL context to use for saving."""
+        """
+        Sets the SQL context to use for saving.
+        .. note:: Deprecated in 2.1, use session instead.
```

Comment:
If we remove it from Scala in 2.2, then we have to remove it here in 2.2 right? I guess we should note it.